sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.net6.0.cs

```diff
@@ -1998,6 +1998,7 @@ public static partial class BlobClientBuilderExtensions
         public static Azure.Core.Extensions.IAzureClientBuilder<Azure.Storage.Blobs.BlobServiceClient, Azure.Storage.Blobs.BlobClientOptions> AddBlobServiceClient<TBuilder>(this TBuilder builder, string connectionString) where TBuilder : Azure.Core.Extensions.IAzureClientFactoryBuilder { throw null; }
         public static Azure.Core.Extensions.IAzureClientBuilder<Azure.Storage.Blobs.BlobServiceClient, Azure.Storage.Blobs.BlobClientOptions> AddBlobServiceClient<TBuilder>(this TBuilder builder, System.Uri serviceUri) where TBuilder : Azure.Core.Extensions.IAzureClientFactoryBuilderWithCredential { throw null; }
         public static Azure.Core.Extensions.IAzureClientBuilder<Azure.Storage.Blobs.BlobServiceClient, Azure.Storage.Blobs.BlobClientOptions> AddBlobServiceClient<TBuilder>(this TBuilder builder, System.Uri serviceUri, Azure.Storage.StorageSharedKeyCredential sharedKeyCredential) where TBuilder : Azure.Core.Extensions.IAzureClientFactoryBuilder { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Binding strongly typed objects to configuration values is not supported with trimming. Use the Configuration Binder Source Generator (EnableConfigurationBindingGenerator=true) instead.")]
```

Comment:
This file is actually generated by our eng/ExportsAPI.ps1 script.

This change is fine, I just wonder if there's a way to have this be generated every single time that way this doesn't get erased
@seanmcc-msft @jsquire @JoshLove-msft Could anyone chime in on this? I might not know the right person for this.

Reply:
This is just the API listing file used for verifying that we don't make breaking changes between versions.    There shouldn't be a need to add anything here directly.